libs/community/langchain_community/llms/ipex_llm.py

```diff
@@ -10,22 +10,22 @@
 logger = logging.getLogger(__name__)
 
 
-class BigdlLLM(LLM):
```

Comment:
for backwards compatibility can we keep around a bigdl.py file and does 
```python
from langchain_community.llms.ipex_llm import IPEXLLM

BigdlLLM = IPEXLLM
```

that way existing code that uses this integration won't break

Reply:
Thanks for the comment. I could not use `BigdlLLM = IPEXLLM` as the imports are different. Instead I inherite from IpexLLM and override necessary methods.